manifests/poc-data-cluster/prod/spark/hip88-silver.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hip88-silver-query
+  namespace: spark
+data:
+  query.sql: |
+    WITH raw AS (
+      SELECT
+        distinct gateway
+      FROM packet_router_packet_report_v1
```

Comment:
I think we want a more generally usable packet router packet report silver table. We'll do the select distinct in another query that is unioned with PoC data

Reply:
Removed file